Gopkg.toml

```diff
@@ -27,7 +27,7 @@ ignored = ["google.golang.org/appengine*"]
   name = "golang.org/x/text"
 
 [[constraint]]
-  revision = "f2499483f923065a842d38eb4c7f1927e6fc6e6d"
```

Comment:
Is it ok to tag a branch, won't `dep ensure` CI check go to latest commit of branch?

Reply:
Dep ensure will follow the Gopkg.lock. The constraint is use for dep ensure -update or dep status to report new commit.
Exemple with git and sdk:
```
PROJECT                                        CONSTRAINT          VERSION        REVISION  LATEST   PKGS USED
code.gitea.io/git                              branch master       branch master  6ef79e8   f3afe30  1   
code.gitea.io/sdk                              branch master       branch master  021567c   2d00f9f  1   
```
We are not at latest of master currently but `dep ensure` doesn't complain.